src/pages/components/externalPages/ExternalPageTemplate.js

```diff
@@ -3,11 +3,14 @@ import { Link } from 'react-router-dom';
 import { LazyLoadImage } from 'react-lazy-load-image-component';
 import FooterContent from '../footerSection/FooterContent';
 import CookiePolicy from '../CookiePolicy';
+import useDarkMode from '../../../utilities/hooks/useDarkMode';
 
 const ExternalPageTemplate = ({ title, children }) => {
+  useDarkMode();
+
```

Comment:
```suggestion
  const mode = useDarkMode();

```

Pls add this so it will be reflected also in external pages. Then pass that as prop to FooterContent component below

```
<FooterContent mode={mode} />
```

Reply:
Because I notice the footer was not change in any external pages when changed to dark mode